apps/console/src/layouts/dashboard-layout.tsx

```diff
@@ -36,7 +36,7 @@ import {
 } from "@wso2is/admin.core.v1";
 import { CommonUtils as ConsoleCommonUtils } from "@wso2is/admin.core.v1/utils";
 import { applicationConfig } from "@wso2is/admin.extensions.v1";
-import { FeatureGateConstants } from "@wso2is/admin.extensions.v1/components/feature-gate/constants/feature-gate";
+import { FeatureGateConstants } from "@wso2is/admin.feature-gate.v1/constants/feature-gate";
```

Comment:
Shall we update the exports to default exports with this effort?

Reply:
+1

Fixed with https://github.com/wso2/identity-apps/pull/6817/commits/84c8547766113dd0c709c5f83b497bb779cfae34